client/blocks/reader-full-post/_content.scss

```diff
@@ -442,3 +442,70 @@
 	padding-left: 16px;
 	padding-right: 16px;
 }
+
+// Writing Prompts Block Styles
+.reader-full-post__story-content .jetpack-blogging-prompt {
+	border: 1px solid var(--color-neutral-5);
+	border-radius: 2px;
+	box-sizing: border-box;
+	margin-bottom: 24px;
+	padding: 24px;
+
+	.jetpack-blogging-prompt__answers {
+		display: flex;
+
+		.image-wrapper {
+			&:first-child {
+				margin-left: -2px;
+			}
+			&:not(:first-child) {
+				margin-left: -15px;
+			}
+
+			img {
+				border: 2px solid #fff;
+				border-radius: 50%;
+				height: 24px;
+				width: 24px;
+			}
+		}
+	}
+
+	.jetpack-blogging-prompt__answers-link {
+		color: var(--studio-gray-80);
+		font-family: sans-serif;
+		font-size: $font-body;
+		margin-left: 6px;
+		text-decoration: underline;
+
+		&:hover {
+			text-decoration: none;
+		}
+	}
+
+	.jetpack-blogging-prompt__label {
+		font-family: sans-serif;
+		font-size: $font-body-small;
+		margin-bottom: 16px;
+		padding-left: 22px;
+		position: relative;
+
+		&::before {
+			background: no-repeat url(https://s0.wp.com/wp-content/mu-plugins/jetpack-plugin/production/_inc/blocks/images/icon-cd4e9ecf53fadf95246d.svg);
```

Comment:
On the mobile app readers this is not looking great in Dark Mode, since this has a very dark gray color. I wonder if there's any way to tint the icon programmatically using CSS in this case. If there is, we (mobile team) will need to add some special treatment here inside the Reader renderers.

Here's an example of what that looks like on Android and iOS:
Android | iOS
--- | ---
<img src="https://user-images.githubusercontent.com/5091503/233734496-62cd4254-8883-4555-91ee-b6a18edf665f.png" height="640"> | <img src="https://user-images.githubusercontent.com/5091503/233734586-f110b00f-ef49-4c7e-9abd-69bab5e03333.PNG" height="640">

**Note:** For supporting dark and light themes we already explicitly set some colors for the text and other things, so I believe it would make sense to add the special treatment on the mobile Reader renderers, using CSS ideally, though I have no idea how to tint icons like this.

Reply:
@thomashorta is there any CSS class to denote that you are in dark mode?

This particular icon is loaded from an SVG file, which means that you won't be able to alter the stroke or fill dynamically. The SVG would need to be included in the DOM for it to be editable (which we can't do here, since we're styling code that is coming from the editor).

What I have done however is include a dark mode variation located at 'client/assets/images/icons/light-bulb-dark.svg'. You could just swap this out with CSS with something like:

```
.dark-mode .jetpack-blogging-prompt .jetpack-blogging-prompt__label::before {
	background: no-repeat url(calypso/assets/images/icons/light-bulb-dark.svg);
}
```

Assuming there is a `.dark-mode` class or something similar that you could use.

Reply:
Not sure why, but when targeting the latest `Calypso Live` URL the icon is not showing up anymore on mobile (both iOS and Android).

Opening the reader on the web it shows up correctly, though I noticed that when I inspect the CSS of the web page, this is the URL being used for the light-bulb: `url(/calypso/images/light-bulb-5a92e3ed7ded3d7f7cae.svg)`

And on mobile I see `url(calypso/assets/images/icons/light-bulb.svg)`.

Looking at the proxy it also seems like it's trying to reach `container-recursing-raman.calypso.live/calypso/calypso/assets/images/icons/light-bulb.svg`, but that returns a `404`. 🤔 

Do you know what could be going ont?

Reply:
> is there any CSS class to denote that you are in dark mode?
Not really, on Android we actually compute the colors on the native site and provide the variables manually for the current App Theme setting. 

On iOS I see they are using `@media (prefers-color-scheme: light)` and `@media (prefers-color-scheme: dark)` to provide the colors, so _maybe_ that works on Android as well... I could not test with the current icons because of the error [I mentioned above](https://github.com/Automattic/wp-calypso/pull/76092#discussion_r1175820457).

Reply:
in https://github.com/Automattic/wp-calypso/pull/76092/commits/0f7116bef0c58a61e97d378c92d7ba6ef20eca2d I moved the icons inline (therefore eliminating the need to load them externally). I also added the code to update the icon when dark mode is selected. Have a look and let me know if this is still posing any issues.

Reply:
> I see we are using @media (prefers-color-scheme: light) and @media (prefers-color-scheme: dark) to provide the colors, so maybe that works on Android as well.

Although it works on mobile, I just realized that it also affects web, which is not great since people can be in dark mode (for instance I use MacOS dark mode) but the reader page on web always has a white background. I did a quick test editing the CSS for a page in Chrome and using the `prefers-color-scheme` and it applies the `dark` variation for me.

So I guess this can't be used on the general CSS, maybe we will have to do something on the mobile clients anyway.

https://user-images.githubusercontent.com/5091503/234290590-d80618dd-6359-4ecf-a6e1-802cf96a5db2.mov